deployment/base/scripts/init_data/03.UserRole.ini

```diff
@@ -75,3 +75,13 @@ partnerId=0
 status=1
 permissionNames="PLAYBACK_BASE_PERMISSION"
 identifierParam="strId"
+
```

Comment:
Please create migration script to execute this ini file and add all migration scripts to the release-notes.